test/datagateway_api/icat/test_icat_client.py

```diff
@@ -10,6 +13,44 @@ def test_init(self):
 
         assert not test_icat_client.autoLogout
 
+    @pytest.mark.parametrize(
+        "client_use, expected_url, expected_check_cert",
+        [
+            pytest.param(
+                "datagateway_api",
+                "https://localhost:8181/ICATService/ICAT?wsdl",
+                False,
+                id="DataGateway API Usage",
+            ),
+            pytest.param(
+                "search_api",
+                "https://localhost.testdomain:8181/ICATService/ICAT?wsdl",
+                True,
+                id="Search API Usage",
+            ),
+        ],
+    )
+    def test_client_use(
+        self, test_config, client_use, expected_url, expected_check_cert,
+    ):
+        class MockClient:
+            def __init__(url, checkCert=True):  # noqa
+                print(f"URL: {url}, Cert: {checkCert}")
```

Comment:
Just repeating my second question from my previous review comment. Is the `print` statement here needed?
```suggestion
```

Reply:
Oh I thought the original commit suggestion deleted it, thanks for rechecking